PizzaAPI/PizzaAPI/Controllers/CommentsController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using PizzaAPI.Models;
+using PizzaAPI.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using PizzaAPI.Helpers;
+
+namespace PizzaAPI.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class CommentsController : ControllerBase
+    {
+        private readonly ICommentsRepository _repository;
+
+        public CommentsController(ICommentsRepository repository)
+        {
+            _repository = repository;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Comments>>> GetPizzas()
+        {
+            var pizzas = await _repository.GetAll();
```

Comment:
Zasadniczo jeśli nie jest to oczywiste jaki typ jest zwracany, to lepiej unikać `var` 
```suggestion
            IEnumerable<Comments>pizzas = await _repository.GetAll();
```